src/devices/cpu/tlcs900/tmp94c241.h

```diff
@@ -0,0 +1,419 @@
+// license:BSD-3-Clause
+// copyright-holders:AJR,Wilbert Pol,Felipe Sanches
+/****************************************************************************
+
+    Toshiba TMP94C241 microcontroller
+
+****************************************************************************/
+
+#ifndef MAME_CPU_TLCS900_TMP94C241_H
+#define MAME_CPU_TLCS900_TMP94C241_H
+
+#pragma once
+
+#include "tlcs900.h"
+
+#define TMP94C241_INTE45   0
+#define TMP94C241_INTE67   1
+#define TMP94C241_INTE89   2
+#define TMP94C241_INTEAB   3
+#define TMP94C241_INTET01  4
+#define TMP94C241_INTET23  5
+#define TMP94C241_INTET45  6
+#define TMP94C241_INTET67  7
+#define TMP94C241_INTET89  8
+#define TMP94C241_INTETAB  9
+#define TMP94C241_INTES0   10
+#define TMP94C241_INTES1   11
+#define TMP94C241_INTETC01 12
+#define TMP94C241_INTETC23 13
+#define TMP94C241_INTETC45 14
+#define TMP94C241_INTETC67 15
+#define TMP94C241_INTE0AD  16
+#define TMP94C241_INTNMWDT 17
+
+// Flip-Flop operations:
+#define FLIPFLOP_INVERT   0b00
+#define FLIPFLOP_SET      0b01
+#define FLIPFLOP_CLEAR    0b10
+#define FLIPFLOP_DONTCARE 0b11
```

Comment:
Please avoid polluting the global namespace in headers, and particularly avoid unnecessary use of preprocessor macros.

Reply:
Here I am following the existing code-style of the other TLCS900 variants in MAME:
https://github.com/mamedev/mame/blob/9c36794ded21e8869b50041981676b69f4a64412/src/devices/cpu/tlcs900/tmp95c061.cpp#L152-L162

Reply:
Do you accept the usage of the same code-style as the other TLCS900 variants already in MAME to be used in this pull request as well?

If that's not acceptable, then what would be your suggestion of a different approach to name these MCU registers in the source?

Reply:
@cuavas, what about using `constexpr u8 TMP95C061_INTE45 = 0x71;` here?

Reply:
Or even just `constexpr u8 INTE45 = 0x71;` (without the prefix indicating the specific CPU model)